src/openms/source/ANALYSIS/MRM/ReactionMonitoringTransition.cpp

```diff
@@ -284,4 +284,36 @@ namespace OpenMS
     library_intensity_ = intensity;
   }
 
+  bool ReactionMonitoringTransition::isDetectingTransition() const
+  {
+    bool detecting = true;
+    if (this->getMetaValue("detecting_transition").toString() == "0") {detecting = false;}
```

Comment:
I would store a boolean value ("true" / "false" as string in the XML file). If somebody else reads the xml, s/he might not know what 0/1 means and true/false is much clearer